components/SideDrawer.tsx

```diff
@@ -0,0 +1,67 @@
+import {
```

Comment:
There are quite a few unused imports in this file. Could you please remove them?